ebpf/redirect.bpf.c

```diff
@@ -125,19 +125,16 @@ authorize_v4(bpf_sock_addr_t *ctx)
             return BPF_SOCK_ADDR_VERDICT_PROCEED;
         }
 
-        // if (ctx->msg_src_ip4 == 0)
-        // {
-        //     bpf_printk("Local/source ip is not set, redirect to loopback ip.");
-        //     ctx->user_ip4 = policy->destination_ip.ipv4;
-        // }
-        // else
-        // {
-        //     ctx->user_ip4 = ctx->msg_src_ip4;
-        //     bpf_printk("Local/source ip is set, redirect to source ip:%u.", ctx->user_ip4);
-        // }
-
-        bpf_printk("redirecting to destination loopback ip.");
-        ctx->user_ip4 = policy->destination_ip.ipv4;
+        if (ctx->msg_src_ip4 == 0)
```

Comment:
seems like the auto-merge revert my changes.

Reply:
https://github.com/Azure/GuestProxyAgent/commit/13dc191ddf1e911e376485655d5fa762a2acb9f1#diff-c2fd1f1dcf93f19078fc1edf1937906ab6e5315a93d86b7283edcf344fbadcff

Reply:
Ack, will revert these changes.